tools/golang/Dockerfile

```diff
@@ -42,4 +42,10 @@ CMD ["golangci-lint", "run", "./..."]
 
 FROM base AS unit
 
-CMD sh -c 'go test $(go list ./... | grep -v  -e experimental -e "test/router" -e "api/v14")'
+VOLUME ["/junit"]
+
+RUN go get github.com/wadey/gocovmerge &&\
+    go get -u github.com/jstemmer/go-junit-report &&\
+    go get -v ./...
```

Comment:
do you need to rerun`go get ./...` here since it was run in the base layer?

Reply:
Was it? I didn't realize. Thought the linter ran that. I can take this out.

Reply:
yep! the base layer runs it